projects/packages/search/src/dashboard/class-dashboard.php

```diff
@@ -47,7 +47,8 @@ class Dashboard {
 	 * @param Automattic\Jetpack\Search\Module_Control $module_control - Module_Control instance.
 	 */
 	public function __construct( $plan = null, $connection_manager = null, $module_control = null ) {
-		$this->plan               = $plan ? $plan : new Plan();
+		$this->plan = $plan ? $plan : new Plan();
+		// TODO: 'jetpack-search' better to be the current plugin where the package is running.
 		$this->connection_manager = $connection_manager ? $connection_manager : new Connection_Manager( 'jetpack-search' );
```

Comment:
I'm not quite understanding your comment here.

Do you mean we'd want to change `'jetpack-search'` to reflect which plugin we're running in? (E.g. `'jetpack-search'` in the standalone plugin, `'jetpack'` in the mainline plugin?)

Reply:
Yes, you are right - sorry for the poor expression. I guess we'll need a flag or something to mark which env it is in.

Reply:
That makes sense, thanks for the clarification 🙂